### ISSUE TITLE
intro-to-rnn: tf.squeeze function, parameter squeeze_dims is deprecated in TF 1.0

### ISSUE THREAD

**Issue by e7dud7e** [udacity/dlnd-issue-reports#197]
Since the intro-to-rnn notebook requires TensorFlow 1.0, the TF 1.0 documentation points out that for tf.squeeze function, the squeeze_dims parameter is deprecated, and replaced by axis.  This tf.squeeze function is being used in the build_rnn function, and is setting the deprecated squeeze_dims parameter.  I recommend replacing:
```
rnn_inputs = [tf.squeeze(i, squeeze_dims=[1]) for i in tf.split(x_one_hot, num_steps, 1)]
```
With this:
```
rnn_inputs = [tf.squeeze(i, axis=[1]) for i in tf.split(x_one_hot, num_steps, 1)]
```

[tf.squeeze for TF 1.0] (https://www.tensorflow.org/api_docs/python/tf/squeeze)

**Comment by mcleonard** [comment]
Just changed this to use tf.nn.dynamic_rnn
